cli/core/products/models/template.py

```diff
@@ -35,7 +35,10 @@ def from_dict(cls, data: dict[str, Any]) -> Self:
             type=data[constants.TEMPLATES_TYPE]["value"],
             content=data[constants.TEMPLATES_CONTENT]["value"],
             content_coordinate=data[constants.TEMPLATES_CONTENT]["coordinate"],
-            default=data[constants.TEMPLATES_DEFAULT]["value"] == "True",
+            default=(
+                None if data[constants.TEMPLATES_DEFAULT]["value"] is None
+                else data[constants.TEMPLATES_DEFAULT]["value"] == "True"
+            ),
```

Comment:
Just a small suggestion to improve readability

```
default = data[constants.TEMPLATES_DEFAULT]["value"]
return cls(
    id=data[constants.TEMPLATES_ID]["value"],
    coordinate=data[constants.TEMPLATES_ID]["coordinate"],
    action=DataActionEnum(data[constants.TEMPLATES_ACTION]["value"]),
    name=data[constants.TEMPLATES_NAME]["value"],
    type=data[constants.TEMPLATES_TYPE]["value"],
    content=data[constants.TEMPLATES_CONTENT]["value"],
    content_coordinate=data[constants.TEMPLATES_CONTENT]["coordinate"],
    default=None if default is None else default == "True",
)
```

Reply:
And how it is happen that default becomes optional now? I have feeling that in platform default is True or False

Reply:
It's changed for subscriptions templates. Default is disabled for them -> https://softwareone.atlassian.net/wiki/spaces/mpt/pages/5754093754/Subscription+templates